src/components/DetailDrawer.tsx

```diff
@@ -195,6 +199,8 @@ export const DetailDrawer: React.FC<{
       <TableRowCustom>
         {columns.map(column => {
           if (shownColumns.includes(column.rowValueDescriptor)) {
+            const indexableField = field as LooselyIndexableField
```

Comment:
Any recommendations? This was all I could think of to avoid having a `prettier-ignore` and `@ts-ignore` for one line... Which seemed... Wrong.

Reply:
Ugh!

I think this is better

            `const unformattedValue = field[column.rowValueDescriptor as keyof ILookmlModelExploreField ]`

Not great, but better than creating a crappy interim interface

Reply:
TIL... `keyof`... nice